internal/fixture/fixture.go

```diff
@@ -356,6 +356,7 @@ func FixBindingWithInstanceID(bindingID string, instanceID string) internal.Bind
 
 		CreatedAt: time.Now(),
 		UpdatedAt: time.Now().Add(time.Minute * 5),
+		ExpiresAt: time.Now().Add(time.Second * 600),
```

Comment:
Please consider comment, there are people who do not get that 600 seconds is equal to 10 minutes.